internal/diff/diff_test.go

```
@@ -341,6 +332,61 @@ func TestUntrackedStatsLoadLazily(t *testing.T) {
+	if os.Geteuid() == 0 {
+		t.Skip("root reads any file regardless of mode")
+	}
+	driver, dir := testRepo(t)
+	write(t, dir, "tracked.go", "package a\n")
+	commit(t, dir, "init")
+	write(t, dir, "locked.go", "package a\n")
+	locked := filepath.Join(dir, "locked.go")
+	if err := os.Chmod(locked, 0o000); err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() { os.Chmod(locked, 0o644) })
```

golangci-lint (2.12.2)

[error] Error return value of `os.Chmod` is not checked

(errcheck)